examples/preact/authenticated-routes/src/routes/login.tsx

```
@@ -0,0 +1,94 @@
+import { createFileRoute } from '@tanstack/preact-router'
```

ESLint

[error] '@tanstack/preact-router' imported multiple times.

(import/no-duplicates)